omod/src/main/webapp/pages/reports/inpatientList.gsp

```diff
@@ -80,8 +82,10 @@
     %>
     <tr id="visit-${ v.patientId
     }">
-        <td>${ v.zlEmrId ?: ''}</td>
-        <td>${ v.dossierNumber ?: ''}</td>
+        <td>${ v.zlEmrId ?: v.primaryIdentifier }</td>
```

Comment:
I'm pretty sure the zlemrid _is_ the primary identifier in Haiti, right?  Or is this sometimes different?  If it is the same, can't we just use primaryIdentifier everywhere?

Reply:
You are correct, @mseaton ! I just replaced the zlEmrId with the primaryIdentifier. Thanks!